src/enkf/params.f90

```diff
@@ -254,6 +254,11 @@ module params
 ! write ensemble mean analysis (or analysis increment)
 logical,public :: write_ensmean = .false.
 
+! taper analysis ens perturbations at top of model (gfs only)
+logical, public :: taperanalperts = .false.
+real(r_kind), public :: taperanalperts_akbot = 500.
```

Comment:
Replace `500.` with `500.0_r_kind` and `-1.` with `-1.0_r_kind` on the line below.